@shared/helpers/soroban/server.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  Transaction,
+  Memo,
+  MemoType,
+  Operation,
+  Server,
+  xdr,
+  scValToNative,
+} from "soroban-client";
+import { captureException } from "@sentry/browser";
+
+export const simulateTx = async <ArgType>(
+  tx: Transaction<Memo<MemoType>, Operation[]>,
+  server: Server,
+): Promise<ArgType> => {
+  const { results } = await server.simulateTransaction(tx);
+  if (!results || results.length !== 1) {
+    throw new Error("Invalid response from simulateTransaction");
+  }
+  const result = results[0];
+  const scVal = xdr.ScVal.fromXDR(result.xdr, "base64");
+  let convertedScVal: any;
+  try {
+    // handle a case where scValToNative doesn't properly handle scvString
+    convertedScVal = scVal.str().toString();
+    return convertedScVal;
```

Comment:
Ah, I _believe_ this should be fixed now in the last soroban-client. So I don't think you need to handle this scvString case

Reply:
oh yeah I did hear that also, I'll test it out to make sure.

Reply:
done in 1ba0450fb6a466570f2ab0318b838f1072f0ae47